src/screens/LandingScreen/LandingScreen.scss

```diff
@@ -0,0 +1,612 @@
+.landing-block {
```

Comment:
why did you decided to rewrite styles to scss ?

Reply:
Easy to me to modify SCSS, i have plugin in VS Code with tips for CSS props. I moved back to PCSS